app/routes/application.js

```diff
@@ -1,84 +1,97 @@
-import Ember from 'ember';
-import config from '../config/environment';
-import preloadDataMixin from '../mixins/preload_data';
+import Ember from "ember";
+import config from "../config/environment";
+import preloadDataMixin from "../mixins/preload_data";
 const { getOwner } = Ember;
 
 export default Ember.Route.extend(preloadDataMixin, {
-
   i18n: Ember.inject.service(),
   isErrPopUpAlreadyShown: false,
   isItemUnavailable: false,
   isLoginPopUpAlreadyShown: false,
   logger: Ember.inject.service(),
   messageBox: Ember.inject.service(),
   cordova: Ember.inject.service(),
-  isMustLoginAlreadyShown: false,
 
-  _loadDataStore: function(){
-    return this.preloadData().catch(error => {
-      if (error.status === 0 || (error.errors && error.errors[0].status === "0")) {
-        this.transitionTo("offline");
-      } else {
-        this.handleError(error);
-      }
-    }).finally(() => {
-      // don't know why but placing this before preloadData on iPhone 6 causes register_device request to fail with status 0
-      if (this.session.get('isLoggedIn')) {
-        this.get("cordova").appLoad();
-      }
-    });
+  _loadDataStore: function() {
+    return this.preloadData()
```

Comment:
should we maybe increase the line length of prettier ? this is horrible 😄

Reply:
yeah, i think its configurable.